src/main/kotlin/no/nav/syfo/util/RequestUtil.kt

```diff
@@ -25,4 +25,14 @@ fun PipelineContext<out Unit, ApplicationCall>.getPersonidentHeader(): String? {
     return this.call.request.headers[NAV_PERSONIDENT_HEADER]
 }
 
+suspend fun PipelineContext<out Unit, ApplicationCall>.withValidToken(
+    block: suspend PipelineContext<out Unit, ApplicationCall>.() -> Unit
+) {
+    if (getBearerHeader() != null) {
+        block()
+    } else {
+        throw IllegalArgumentException("No Authorization header supplied")
+    }
+}
+
```

Comment:
Tanken er at man i stedet for å gjøre 
```kotlin
val token = getBearerHeader() ?: throw IllegalArgumentException("No Authorization header supplied")
```
og ikke bruke `token` til noe, så kan man wrappe kallet i en slik og gjøre sjekken der 👌🏼

Reply:
Det ble fint! 🏆